src/lesson9/HomemadeArrayList/hArrayList.java

```diff
@@ -0,0 +1,129 @@
+package lesson9.HomemadeArrayList;
+
+public class hArrayList implements hList {
```

Comment:
class name starting with lower case, need to be UPPERCASE,  and class name need to have some mining